docs/how-to/configure.md

```diff
@@ -98,6 +98,15 @@ The checks take 24 hours to complete.
 
 The results can be viewed in *PMM > PMM Database Checks*.
 
+**DBaaS**
```

Comment:
this flag is visible only if you pass the parameter, so it's in the early Tech  Preview stage. we need to mention this